extensions/channels.py

```diff
@@ -63,10 +56,7 @@ async def thread(self, ctx, title, category: converters.CategoryConverter, *, co
             description=f"A new thread has been started in {channel.mention} by {ctx.author.mention}.",
             color=discord.Color(0x007fff)))
 
-        conn = self.bot.conn
-        c = conn.cursor()
-        c.execute("INSERT INTO threads VALUES (?, ?)", (channel.id, ctx.author.id))
-        conn.commit()
+        await self.bot.database.create_thread(ctx.channel.id, ctx.author.id)

```

Comment:
This line errors:
```
Ignoring exception in command thread:
Traceback (most recent call last):
  File "/home/object/.local/lib/python3.7/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/object/GuildBot-dev/extensions/channels.py", line 60, in thread
    await ctx.bot.database.create_thread(ctx.channel.id, ctx.author.id)
  File "/home/object/GuildBot-dev/utils/database.py", line 23, in create_thread
    async with self.pool.acquire() as conn:
AttributeError: 'NoneType' object has no attribute 'acquire'
```

Reply:
Could not reproduce. Seems to be that `create_pool` call failed for some reason and returned None. Potentially fixed in 6120eac91f79182e9a33a717430aaba11be31303